pulsar/client/staging/down.py

```diff
@@ -143,7 +143,7 @@ def _attempt_collect_output(self, output_type, path, name=None):
         # name is the 'name' of the file on the Pulsar server (possible a relative)
         # path.
         collected = False
-        with self.exception_tracker():
+        with self.exception_tracker(allow_failure=_allow_collect_failure(output_type)):
             action = self.action_mapper.action({"path": path}, output_type)
             if self._collect_output(output_type, action, name):
```

Comment:
Can you move this behavior into collect_output for these outputs? This seems like an odd level to handle it at right? Like - if this isn't an exception condition don't throw it as one?